lib/execution_engine2/db/MongoUtil.py

```diff
@@ -263,6 +263,33 @@ def check_if_already_finished(job_status):
             return True
         return False
 
+    def update_jobs_to_queued(self, job_ids, scheduler_ids):
+        bulk_operations = []
+        now = time.time()
+        for i, job_id in enumerate(job_ids):
+            bulk_operations.append(
+                UpdateOne(
+                    {"_id": ObjectId(job_id)},
+                    {
+                        "$set": {
+                            "status": Status.queued.value,
+                            "queued": now,
+                            "scheduler_id": scheduler_ids[i],
+                            "scheduler_type": "condor",
```

Comment:
This is coupling MongoUtils to Condor - it might make more sense to pass this in, either in the constructor or in the method

Reply:
Seems like a small amount of code to change if and when we decide to change schedulers, doesn't seem worth it right now to pass it in.

Reply:
[Reducing code coupling is a good thing in general](https://en.wikipedia.org/wiki/Coupling_(computer_programming)), and I mean, this is a 5 min fix so I'd just do it

Reply:
We removed the entire scheduler class to inherit from, so I am assuming we aren't going to have any new schedulers anytime soon, seems like YAGNI to me.

Reply:
If you make a class with `run_job`, `get_job_resource_info`, and `cancel_job` methods you've just made a new scheduler class that you can plug right in with ducktyping, so I don't think removing the abstract class really affects things. It'd be easy to make a new one based on the above methods if we wanted to anyway.

YAGNI is for stuff like features and optimization, not code quality issues like cohesion and separation of concerns. You always want to do that when possible.

I'm honestly a little confused why this this so controversial - we've spent more time talking about it than it would have taken to just make the change at this point. Given the change is so simple, what's your concern about making it?

Reply:
Oh, according to https://github.com/kbase/execution_engine2/pull/406#discussion_r672770748 you did make the change. I'll take a look at the diff in a sec

Reply:
:+1: